### DISCUSSION TITLE
Support pgvectorscale for better scalability compared to Pinecone

### DISCUSSION THREAD

**Discussion by da7a90-backup** [supabase/supabase#27474] [Feature Requests]
pgvectorscale was developed by the timescale team and according to their benchmarks using it in conjunction with pgvector exponentially increases the performance when it comes to speed and accuracy. I don't really see a way of integrating it into our DB directly through the SQL editor as a custom extension so I'm posting here hoping it would get added soon since I don't think it would be a difficult integration. Thanks!

**Comment by GaryAustin1** [comment]
It seems like it is being worked on:
https://x.com/avthars/status/1800682346162831424

**Reply by octalpixel** [reply]
Is there any update on this by anychance

**Reply by da7a90-backup** [reply]
doesn't seem like it

**Reply by quantpioneer** [reply]
Any update on integrating `pgvectorscale`?

**Reply by lukapmoran** [reply]
I'd also be interested in implementing this if it become supported.

**Reply by misterdojo777** [reply]
any update ?

**Reply by quantpioneer** [reply]
We neeeeeed it

**Comment by le-dawg** [comment]
So, any updates on this? We are considering supabase in all our projects, but for this new demanding client we do need SPEED.

**Comment by moyindavid16** [comment]
Bumping this up. This would be a useful extenion added to supabase

**Comment by anshunjain** [comment]
bumping this up. it would really eliminate the need to go to pinecone for our use cases
